templates/repo/view_list.tmpl

```diff
@@ -34,7 +34,11 @@
 					</span>
 				{{end}}
 			</th>
-			<th class="text grey right age">{{if .LatestCommit}}{{if .LatestCommit.Committer}}{{TimeSince .LatestCommit.Committer.When $.i18n.Lang}}{{end}}{{end}}</th>
+			<th class="text grey right age">
+				<span class="mobile-align">
```

Comment:
`mobile-align` is not a proper name , this CSS class is also rendered in desktop browsers (although the styles in it are empty).

Reply:
https://github.com/go-gitea/gitea/pull/19515/commits/21ceac153912026593f2120e5ab8ea514935fb8b